test/integration/examples/custom/helloworld/helloworld_podman_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build (podman && cli) || (podman && examples) || (podman && integration)
+//go:build (podman && cli) || (podman && integration)
```

Comment:
I believe this one should use: `podman && examples` instead, right?

Reply:
I was planning to leave the HelloWorld examples (with and without podman) in the normal build; I think that the podman tests need to be executed just in case in every build. What do you think?

Reply:
Makes sense.
What about moving them out of the test/integration/examples/custom into
test/integration/acceptance/custom ?

Reply:
good idea!